### DISCUSSION TITLE
Pycharm works fine but not that good on neovim how to solve it?

### DISCUSSION THREAD

**Discussion by OriLiMu** [community/community#35219] [Copilot Conversations]
# Nothing happend on neovim in this case ( somethings it works fine but usually copilot on neovim not that good as pycharm)

![image](https://user-images.githubusercontent.com/11295355/194124098-cd9b75c8-e7b5-4a84-a5d0-aff15a36de45.png)

# Result on pycharm
![image](https://user-images.githubusercontent.com/11295355/194123767-5ece8111-363a-4c5f-854e-3464123457fa.png)

Why this happens and how to fix it?

**Comment by iamcjwilliams** [comment]
@woshichuanqilz Our copilot team is reviewing this issue

**Comment by kimadeline** [comment]
Hi @woshichuanqilz, which languages have the worst/best experience in Neovim? What does `:Copilot status` say when you have no suggestions? What does your Copilot configuration look like?

**Comment by tpope** [comment]
The Vim plugin disables Copilot in Markdown by default. If you'd like to override that, add this to your vimrc:

```vim
let g:copilot_filetypes = {'markdown': v:true}
```

Or this to your init.lua:

```lua
vim.g.copilot_filetypes = {markdown = true}
```
